README-Localized/README-es-es.md

```diff
@@ -0,0 +1,152 @@
+[![identificador de versión npm](https://img.shields.io/npm/v/@microsoft/microsoft-graph-types.svg)](https://www.npmjs.com/package/@microsoft/microsoft-graph-types)
+
+# Tipos de TypeScript de Microsoft Graph
+Las definiciones de TypeScript de Microsoft Graph permiten a los editores ofrecer IntelliSense en objetos de Microsoft Graph, como usuarios, mensajes y grupos.
+
+## Instalación
+
+Se recomienda incluir el archivo .d.ts al descargar este paquete a través de [npm](https://www.npmjs.com/).
+
+```bash
+
+# Install types and save in package.json as a development dependency
```

Comment:
asking because I am not familiar with the localization process: Is there a way to get translations for comments in the code sections as well?